CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Sorted by name order of columns for `table` and `ttable` formats.
 - `tt switch tt`: does not work with `x.x.x` version format.
+- Added ability to install latest version of `master` tarantool.
```

Comment:
We already have `Added` a couple of lines at top.

```suggestion
- Ability to update a latest version of `master` tarantool with `tt install`.
```

Reply:
Changed it.